File: front/src/models/comment.ts
export interface CommentFull {
    id: number;
    username: string;
    comment: string[];
    userId: number;
    movieId: number
  }

  export interface Comment {
    username: string;
    createdAt: Date;
    comment: string[];
    movieId: number
    userId: number
  }

File: back/comment-service/src/main/java/com/ro/model/Comment.java
package com.ro.model;

import lombok.*;
import javax.persistence.*;
import java.time.LocalDateTime;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "comment")
public class Comment {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private LocalDateTime createdAt;
    private String username;

    @Column(length = 1000)
    private String comment;
    private Long movieId;
    private Long userId;
}